polaris-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -85,6 +91,11 @@ export function Tooltip({
     setTrue: handleFocus,
     setFalse: handleBlur,
   } = useToggle(Boolean(originalActive));
+
+  const {value: persist, toggle: togglePersisting} = useToggle(
+    Boolean(originalActive) && Boolean(persistOnClick),
```

Comment:
Do we need the `persistOnClick` in the Boolean constructor?

Reply:
We do! Otherwise we have persist working when `originalActive` is true at the start.